mtags/src/main/scala/scala/meta/internal/mtags/ScalaMtags.scala

```diff
@@ -283,37 +285,62 @@ class ScalaMtags(val input: Input.VirtualFile, dialect: Dialect)
             }
           }
         case t: Defn.ExtensionGroup =>
-          // t.params are ignored - don't know which symbol/owner they should have
-          // need to wait for https://github.com/lampepfl/dotty/issues/11690
+          val extensionParamss = t.paramss.flatten.map(p => p.name)
           val (owner, overloads) =
             if (isPackageOwner)
               topleveSourceData
             else
               (currentOwner, new OverloadDisambiguator())
 
-          def addDefnDef(t: Defn.Def): Unit =
+          def addDefnDef(t: Defn.Def): Unit = {
```

Comment:
I'm wondering if it might be fixed with concatenation of def.params with exntesions params.
Smth like:
```scala
case  Defn.ExtensionGroup =>
   val extensionParamss = t.paramss.flatten.map(p => p.name)
   
   def addDefnDef(t: Defn.Def): Unit = {
      disambiguatedMethod(
        t,
        t.name,
        Nil,
        t.params ++ extensionParamss, // register both def and extension params
        Kind.CONSTRUCTOR,
        overloads
      )
  }
  ...
```

Reply:
Works well :+1: https://github.com/scalameta/metals/pull/3814/commits/efd3b5952ea72032e9292decb24fae8d23631ad2